package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brainly-style-guide",
-  "version": "220.5.0",
+  "version": "221.0.0-beta.0",
```

Comment:
Please remember to revert to the version

Reply:
changed